### PR TITLE
gh-123363: Show string value of CONTAINS_OP oparg in dis

### PR DESCRIPTION
**Pull request by Alexandr153**
Result:
```
>>> echo 'a not in  b' | ./python.exe -m dis
  0           RESUME                   0

  1           LOAD_NAME                0 (a)
              LOAD_NAME                1 (b)
              CONTAINS_OP              1 (not in)
              RETURN_VALUE
```

```
>>> echo 'a in  b' | ./python.exe -m dis
  0           RESUME                   0

  1           LOAD_NAME                0 (a)
              LOAD_NAME                1 (b)
              CONTAINS_OP              0 (in)
              RETURN_VALUE
```

* Issue: gh-123363

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Show string value of :opcode:`CONTAINS_OP` oparg in :mod:`dis` output.
```
Comment (picnixz) [inline Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst:1]:
```rst
Show string value of :opcode:`CONTAINS_OP` oparg in :mod:`dis` output.
Patch by [...].
```

If you want, you can also credit yourself (just be sure the end the sentence with a period).

**Review by picnixz — APPROVED** [review]

File: Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Patch by [Alexandr153].
```
Comment (picnixz) [inline Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst]:
```suggestion
Patch by Alexandr153.
```

I'm sorry that my suggestion template was misunderstood :')

Comment (Alexandr153) [reply Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst]: That's okay, this is my first experience with CPython, thank you

Comment (picnixz) [reply Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst]: Don't worry, you're doing great for a first experience!

File: Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst (hunk 1 above)
Comment (Eclips4) [reply Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst:1]: Additionally, you can add yourself to Misc/ACKS

**Review by Eclips4 — APPROVED** [review]
Thanks!

**Review by sobolevn — APPROVED** [review]
Thank you!

**Pull request merged**